octoprint_octorelay/__init__.py

```diff
@@ -310,11 +310,12 @@ def print_started(self):
             settings = self.get_settings_defaults()[index]
             settings.update(self._settings.get([index]))
 
+            relay_pin = int(settings["relay_pin"])
+            inverted = settings['inverted_output']
             autoONforPrint = settings['autoONforPrint']
-            if autoONforPrint:
-                relay_pin = int(settings["relay_pin"])
-                inverted = settings['inverted_output']
```

Comment:
@hcooper , is there any reason for moving these two assignments from the scope of the condition up to the higher scope?

Reply:
Ah, I presume this is the answer from the PR description.

> Move definition ofrelay_pin and inverted so both functions match. [code tidy up]
> makes print_started() follow the same logic of print_stopped()

So, it's for achieving similarity to the implementation of another method, right?